src/amo/components/Home/styles.scss

```diff
@@ -114,11 +128,26 @@
   height: 64px;
   margin-bottom: 5px;
   transform: scale(0.8);
-  width: 100%;
+  width: 35%;
 
   @include respond-to(medium) {
     transform: initial;
   }
+
+  @include respond-to(large) {
+    background-position: center;
+    width: 100%;
+  }
+}
+
+*[dir='rtl'] {
+  .Home-SubjectShelf-list-item .Icon {
+    background-position: right;
+    unicode-bidi: embed;
```

Comment:
I think switching the position here is enough.. you can remove this `unicode-bit: embed`